pkg/lockfile/helpers_test.go

```diff
@@ -41,10 +45,20 @@ func hasPackage(packages []lockfile.PackageDetails, pkg lockfile.PackageDetails)
 	return false
 }
 
+func absPathNoErr(path string) string {
```

Comment:
nit: generally I'd consider calling this something like `absPathOrPanic`, or `mustBeAbsPath` if you want to be a true Go traditionalist 😅 - but that way it gives a stronger indication of what happens if it does error

Reply:
Went with `absPathOrPanic`, thanks for the name suggestion, originally I had `absPathButPanicOnErr` and thought that was way too long haha.